scripts/build.js

```diff
@@ -42,22 +48,27 @@ nextTask("Cleaning up previous build", () => {
   );
 });
 
+// TODO:
+// force installing dependencies is not ideal
+// rewrite to ask for permission for each dependency that isnr present or just throw error
+// add option for user to add prettier config
+// reading from package json for some reason does not block other processes
 nextTask("installing dependencies", async () => {
```

Comment:
please take a look at this task, this is rewriting package.json which firstly I don;t think is supposed to be done, secondly if the whole purpose it to add some deps, why not add it to the handlebars and be done with it. Then we will only have to install dependencies here. This task is the one that is causing change in format of package.json. Am I missing something? @SalihuDickson @anuragxxd

Reply:
Sure, I guess that would be a better approach. This was always meant to be a temporary approach and was only viable since it was an internal tool. But you're right, we shouldn't be rewriting package json, I'll fix this.

Reply:
@JaeAeich, #290 fixes this, please review.